apps/site/pages/docs/getting-started/2-setting-up-the-project.mdx

```diff
@@ -165,76 +165,81 @@ Let's make the first level of customization which is update the store's theme wi
 // GLOBAL TOKENS
 // Theme Soft Blue
 // ----------------------------------------------------------
- 
+
+@import '@faststore/ui/src/styles/base/utilities.scss';
+@import '@faststore/ui/src/styles/base/typography.scss';
```

Comment:
Do we need it? since we already import the `global.scss` in `_app.tsx`?
edit: now I see we need. Hmm, I'll create a new task to deal with this 👀

Reply:
Oh, we added it because we were testing the font overrides! But it didn't work as expected, so I think we can remove it.

Reply:
Follow up PR https://github.com/vtex/faststore/pull/1786